src/hooks/useAPIData.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react';
+export const useAPIData = (DataAPI, APIParameter = null) => {
+  const [Data, setData] = useState(null);
```

Comment:
```suggestion
  const [data, setData] = useState(null);
```
통상적으로 파스칼케이스는 클래스에 사용되는 컨벤션이므로 카멜 케이스를 조금 더 권장드립니다.